scripts/time_average.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
+# Licensed under the MIT License
+
+from hera_cal import frf
+
+a = frf.time_average_argparser()
+args = a.parse_args()
+
+frf.time_avg_data_and_write(input_data=args.input_data,
+                            output_data=args.output_data,
+                            t_avg=args.t_avg, rephase=args.rephase,
+                            wgt_by_nsample=not(args.dont_wgt_by_nsample),
```

Comment:
I find this a bit counterintuitive to define this method's argument as the negation of the argument parser's value, but I suppose it makes sense. It might be worth putting a comment either here or in the argparse definition about what's happening.

Reply:
I often do this when I want the user to request the non-default behavior. If we made it --wgt_by_nsample, then the default behavior would be to not wgt by nsample which is not the default behavior in the rest of the code.

Reply:
I've put a little more clarificaion in the argparse string.

Reply:
and added a docstring.